src/Integration/Monolog/CloudWatch/src/CloudWatchLogsHandler.php

```diff
@@ -90,34 +80,35 @@ class CloudWatchLogsHandler extends AbstractProcessingHandler
      *  Log group names can be between 1 and 512 characters long.
      *  Log group names consist of the following characters: a-z, A-Z, 0-9, '_' (underscore), '-' (hyphen),
      * '/' (forward slash), and '.' (period).
-     * @param string $group
-     *
      *  Log stream names must be unique within the log group.
      *  Log stream names can be between 1 and 512 characters long.
      *  The ':' (colon) and '*' (asterisk) characters are not allowed.
-     * @param string $stream
-     * @param int    $batchSize
-     * @param int    $level
-     * @param bool   $bubble
+     * @param array{
+     *   batchSize?: int,
+     *   bubble?: bool,
+     *   group: string,
+     *   level?: int,
```

Comment:
We should not have level and bubble here.

Reply:
indeed, fixed!

thank you